src/flux/performance-export.coffee

```diff
@@ -0,0 +1,73 @@
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+{JobActions, JobStore} = require './job'
+_ = require 'underscore'
+moment = require 'moment'
+
+EXPORTING = 'exporting'
+EXPORT_QUEUED = 'export_queued'
+EXPORTED = 'exported'
+
+PerformanceExportConfig = {
+
+  _job: {}
+
+  getJobIdFromJobUrl: (jobUrl) ->
+    jobUrlSegments = jobUrl.split('/api/jobs/')
```

Comment:
Is parsing the URL necessary or can we just use the `jobUrl` instead of the `jobId`?